src/wp-includes/block-template.php

```diff
@@ -5,6 +5,30 @@
  * @package WordPress
  */
 
+/**
+ * Adds necessary filters to use 'wp_template' posts instead of theme template files.
+ *
+  * @access private
+  * @since 5.9.0
+ */
```

Comment:
```suggestion
/**
 * Adds necessary filters to use 'wp_template' posts instead of theme template files.
 *
 * @access private
 * @since 5.9.0
 */
```